### PR TITLE
[zh-cn]update workloads/ _index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
preview: https://deploy-preview-35246--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/workloads/

### CODE REVIEW

**Comment by windsonsea** [comment]
😄 
The preview link is: 
https://deploy-preview-35246--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/workloads/

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 1]
```diff
@@ -52,10 +52,10 @@ you specified.
 
 Kubernetes provides several built-in workload resources:
 -->
-不过，为了让用户的日子略微好过一些，你并不需要直接管理每个 Pod。
-相反，你可以使用 _负载资源_ 来替你管理一组 Pods。
-这些资源配置 {{< glossary_tooltip term_id="controller" text="控制器" >}}
-来确保合适类型的、处于运行状态的 Pod 个数是正确的，与你所指定的状态相一致。
+不过，为了减轻用户的使用负担，通常不需要用户直接管理每个 Pod。
+而是使用 _负载资源_ 来替用户管理一组 Pods。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/_index.md:56]:
```suggestion
而是使用**负载资源**来替用户管理一组 Pod。
```

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 2]
```diff
@@ -160,8 +157,8 @@ for applications:
 
 * [垃圾收集](/zh-cn/docs/concepts/architecture/garbage-collection/)机制负责在
   对象的 _属主资源_ 被删除时在集群中清理这些对象。
-* [_Time-to-Live_ 控制器](/zh-cn/docs/concepts/workloads/controllers/ttlafterfinished/)
-  会在 Job 结束之后的指定时间间隔之后删除它们。
+* [_Time-to-Live_ 控制器](/zh-cn/docs/concepts/workloads/controllers/ttlafterfinished/)会在 Job
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/_index.md:160]:
```suggestion
* [**Time-to-Live** 控制器](/zh-cn/docs/concepts/workloads/controllers/ttlafterfinished/)会在 Job
```

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 3]
```diff
@@ -25,10 +25,10 @@ a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} wh
 Pod is running means that all the Pods on that node fail. Kubernetes treats that level
 of failure as final: you would need to create a new Pod even if the node later recovers.
 -->
-无论你的负载是单一组件还是由多个一同工作的组件构成，在 Kubernetes 中你
-可以在一组 [Pods](/zh-cn/docs/concepts/workloads/pods) 中运行它。
+在 Kubernetes 中，无论你的负载是由单个组件还是由多个一同工作的组件构成，
+你都可以在一组 [Pods](/zh-cn/docs/concepts/workloads/pods) 中运行它。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/_index.md:29]:
```suggestion
你都可以在一组 [Pod](/zh-cn/docs/concepts/workloads/pods) 中运行它。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
Prefer to use a bold font in zh-cn rather than an italic font.

File: content/zh-cn/docs/concepts/workloads/_index.md (hunk 3 above)
Comment (yanrongshi) [reply content/zh-cn/docs/concepts/workloads/_index.md:29]: Thanks your review,brother

File: content/zh-cn/docs/concepts/workloads/_index.md [hunk 4]
```diff
@@ -37,11 +37,11 @@ For example, once a pod is running in your cluster then a critical fault on the
...
 would need to create a new `Pod` to recover, even if the node later becomes healthy.
 -->
-Kubernetes Pods 有[确定的生命周期](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/)。
-例如，当某 Pod 在你的集群中运行时，Pod 运行所在的
+Kubernetes Pods 遵循[预定义的生命周期](/zh-cn/docs/concepts/workloads/pods/pod-lifecycle/)。
+例如，当在你的集群中运行了某个 Pod，但是 Pod 所在的
 {{< glossary_tooltip text="节点" term_id="node" >}} 出现致命错误时，
-所有该节点上的 Pods 都会失败。Kubernetes 将这类失败视为最终状态：
-即使该节点后来恢复正常运行，你也需要创建新的 Pod 来恢复应用。
+所有该节点上的 Pods 的状态都会变成失败。Kubernetes 将这类失败视为最终状态：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/_index.md:43]: 这个更改不必要。

**Pull request merged**